aiosmtpd/tests/test_handlers.py

```diff
@@ -17,6 +17,19 @@
 CRLF = '\r\n'
 
 
+ModuleResources = ExitStack()
+
+
+def setUpModule():
+    # Needed especially on FreeBSD because socket.getfqdn() is slow on that OS,
+    # and oftentimes (not always, though) leads to Error
+    ModuleResources.enter_context(patch("socket.getfqdn", return_value="localhost"))
+
+
+def tearDownModule():
+    ModuleResources.close()
+
+
```

Comment:
Required to stabilize testing on FreeBSD.

`unittest` seems to be _much_ slower than `pytest` on FreeBSD, don't know why.